sdjournal/read.go

```diff
@@ -244,13 +246,14 @@ process:
 				case SD_JOURNAL_APPEND, SD_JOURNAL_INVALIDATE:
 					continue process
 				default:
-					log.Printf("Received unknown event: %d\n", e)
+					err = fmt.Errorf("received unknown event: %d", e)
+					break process
```

Comment:
It looks like we are using a mix of `break process` and `return err`. As this is removing the named return (:+1:) I think it would make sense to also get rid of the `break`s to make this a bit more readable. What do you think?

Reply:
Thanks, you're right, the code can be further simplified, now! I used to not `defer` the `Wait` and call it manually before the return, but that makes it even nicer. Changing!